src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -83,11 +141,17 @@ class OrganizationDetails extends Component {
                                         :
                                         <span />
                                     }
+
                                     {this.props.accountType === AccountType.RECEIVING_AGENCY ? 
                                         <label className="label-component details">Emergency Pickup</label>
                                         :
                                         <span />
                                     }
+                                    {this.props.accountType === AccountType.RECEIVING_AGENCY ? 
+                                        <label className="label-component details">Availability</label>
+                                        :
+                                        <span />
```

Comment:
Use above suggestion here too, I realize now that the person who wrote all this code originally used empty span tag ternaries but unless I'm missing something the && solution is much cleaner